docs/about/getting-started.md

```diff
@@ -11,24 +11,24 @@ HTTP is the default ingress and is always enabled. The following properties can
 
```

Comment:
this is option 1: use underscore + uppercase

Reply:
I would go with option 1, because it will always work

Reply:
18 days delay in replying :x sorry.
i agree. i'll prepare a pr soon with the changes.